Bottom: b502576d24e1fe51b72b6ba94c906c686d2844af
Top:    019c4c38e0a9c98c0cca32c6aea3c98a9b878c5e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-31 13:19:48 +0100

Refresh of work-in-progress-on-subcomposi

---

diff --git a/src/draw.c b/src/draw.c
index ffdb568..0decdaf 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -521,7 +521,10 @@ DrawEverything (BoxTypePtr drawn_area)
 		  NULL);
       /* Draw rat lines on top */
       if (gui->set_layer ("rats", SL (RATS, 0), 0))
-	DrawRats(drawn_area);
+        {
+          DrawRats(drawn_area);
+          gui->set_layer (NULL, SL (FINISHED, 0), 0);
+        }
     }
 
   for (side = 0; side <= 1; side++)
